docs/usage.md

```diff
@@ -5,77 +5,111 @@
+3. Click **Simulate Scenario**.
 
-    - **Group**, **Tactics**, **Technique**, **Sub-Technique**, and **Software** get linked to alert under `Correlations` tab of Alert.
-    
-    - **Mitigation** get correlated to alert `Recommended ATT&CK Mitigations` tab.
+### OT - Stuxnet Attack Scenario
 
-    - Once the alert has been enriched, open the `Stuxnet peer to peer communication attempt` alert and look for a similar alert in the Workspace `Recommendations` tab. 
+This scenario generates following alerts, each at an interval of 2 seconds:
 
-    - Escalate the alert to an incident by checking `Select All` and `Include this record` then running the playbook **Escalate**. Please provide all incident details. (An incident is created with the link available in the alert's comment)
+- Stuxnet peer-to-peer communication attempt
+- Anomalous communication between two Windows XP machines
+- Communication between public and private networks
+- STEP7 configuration download command
 
-    - Incident get correlated with all the **IOCs**, **Alerts**, **Assets**, **Technique**, **Mitigation**, and **Software**.
+<table>
+    <tr>
+        <td><strong>IMPORTANT</strong></td>
+        <td>Ensure that the global variable <code>Demo mode</code> is set to <code>true</code>.</td>
+    </tr>
+</table>
 
-    - Click on **Quarantine Asset and Raise Ticket** to quarantine assets and raise ticket for assets.
+Navigate to the newly-created alerts and observe the following:
 
-    - You can generate report by using **Generate Incident Summary Report**
-    
-    - Use the `Setup War Room` button to create a war room. All of the alerts, incidents, affected assets, and IOCs will be in the war room.
+- Each asset is created and added to the record with the following hostname:
+
+    - PLC S7-400-324
+    - Dell - 5490-42E6
+    - ThinkCentre M910s-1181
+    - HP - PB7181
+
+- Each alert is associated with assets based on the assets' source and destination IP.
+
+- The **Correlations** tab of an alert links *Group*, *Tactics*, *Technique*, *Sub-Technique*, and *Software*.
+
+- Possible mitigation actions are listed in the alert's **Recommended ATT&CK Mitigation** tab.
+
+- After enrichment completes, the **Stuxnet peer-to-peer communication attempt** alert contains similar alerts in the **Workspace** > **Recommendations** tab. 
+
+    - You can escalate this alert to an incident by selecting **Select All** and **Include this record** checkboxes, click the **Playbook** drop-down and select **Escalate**.
     
-    - Playbooks in the War Room can be used to take action, such as `Isolate Devices From Network`, `Scan All Assets Involved`, and `Update Firewall Policy`.
+    - On the **Escalate** pop-up window, enter relevant details and click the button **Escalate**. An incident is created with the link available in the alert's comment.
+
+- Incidents are correlated with all the *Indicators*, *Alerts*, *Assets*, *Technique*, *Mitigation*, and *Software*.

```

Comment:
Incidents are correlated with all the *Indicators*, *Alerts*, *Assets*, MITRE ATT&CK's **Group**, **Tactics**, **Techniques**, **Sub-techniques**, **Mitigations**, **Software**, and **Mitigation**